packages/zosfiles/src/methods/mount/Mount.ts

```diff
@@ -58,7 +58,7 @@ export class Mount {
 
         const jsonContent = JSON.stringify(tempOptions);
         const headers = [{"Content-Length": jsonContent.length}, ZosmfHeaders.ACCEPT_ENCODING];
-        if (options.responseTimeout != null) {
+        if (options && options.responseTimeout != null) {
```

Comment:
Thanks for adding the check for the Mount APIs
If you are feeling lazy and don't want to type as much, you could also use the `?.` operator
like: `options?.responseTimeout` 

Will adding this extra check decrease branch coverage for this file? 
🤔

Reply:
same code coverage % woot